woocommerce/woocommerce-android
# Add toolbar to PrintShippingLabelInfoFragment

Part of #10570 

### Description

This PR introduces individual toolbars for PrintShippingLabelInfoFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> Print shipping label -> click on "Don't know how to print with your mobile device" text) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="243" alt="Screenshot 2024-02-12 at 6 31 24 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/6ad516ad-5e5f-40e7-8dde-43a499040709">
